### PR TITLE
Jb57092

### PR DESCRIPTION
**Pull request by martyone**

### CODE REVIEW

File: sdk-setup/src/sdk-manage [hunk 1]
```diff
@@ -545,6 +545,13 @@ object_config_get()
     echo "${value:-$default}"
 }
 
+object_config_get_boolean()
+{
+    local value=
+    value=$(object_config_get "$@") || return
+    tr -d '0' <<<$value
+}
```
Comment (vigejolla) [inline sdk-setup/src/sdk-manage:560]: This feels a bit like a hack. But it does work, and it is convenient. And with bash everything feels like a hack. So I guess it's good :)

Comment (martyone) [reply sdk-setup/src/sdk-manage:560]: Yeah after all I did it a more traditional way :)

Comment (martyone) [reply sdk-setup/src/sdk-manage:560]: LOL, I was thinking too fast. Reverted to the hacky way as that allows to separate error handling :)

**Review by vigejolla — APPROVED** [review]
LGTM

**Pull request merged**